### ISSUE TITLE
Merge SSD reference PR's.

### ISSUE THREAD

**Issue by johntran-nv** [mlcommons/training_policies#390]
https://github.com/mlperf/training/pull/421 - Add GA support
https://github.com/mlperf/training/pull/420 - batch eval support
https://github.com/mlperf/training/pull/419 - learning rate compliance
https://github.com/mlperf/training/pull/418 - parametrized several values
https://github.com/mlperf/training/pull/417 - option to load pretrained weights from file
https://github.com/mlperf/training/pull/416 - get default local rank from env variable
https://github.com/mlperf/training/pull/415 - update pytorch to v1.6
https://github.com/mlperf/training/pull/414 - update logging to v0.7

**Comment by bitfort** [comment]
SWG:
To be reviewed by AMD

**Comment by johntran-nv** [comment]
all PRs have been merged or closed now, so closing this issue.
